### DISCUSSION TITLE
plex s1001 network error

### DISCUSSION THREAD

**Discussion by nixon7606** [tteck/Proxmox#3744] [Help Needed]
Is anyone getting this error. Fresh install. only doing Iptv via plex. Plex doesn't seem to CPU transcode at all either. Is there CPU requirements for plex? the CPU on host is Intel(R) Xeon(R) Silver 4210 CPU @ 2.20GHz. I have tried the plex lxc script on all default settings. Also installed on ubuntu and again on debian. all act the same. literally is a fresh install then add a threadfin as a tuner. This setup works perfectly in my windows computer but can't get it to work in proxmox in privileged or unprivileged LXC

**Comment by tteck** [comment]
The Intel® Xeon® Silver 4210 CPU does not have an integrated GPU

**Reply by nixon7606** [reply]
I did pass thru a Nvidia p400 for transcoding. i see the card in the proxmox host and the LXC (nvidia-smi). I have even seen it hardware transcode a few times. 99% of the time I'm getting a s1001 network error. What logs should I look at to troubleshoot? I appreciate any direction that might lead me to a solution.
